### DISCUSSION TITLE
After update from 4.0.11 to 4.1.8: pipeline caused error: wrong number of arguments for 'hset' command

### DISCUSSION THREAD

**Discussion by sbdiun** [netbox-community/netbox#18252] [Help Wanted!]
### Deployment Type

Self-hosted

### Triage priority

N/A

### NetBox Version

v4.1.8

### Python Version

3.10

### Steps to Reproduce

Update Netbox version from 4.0.11 to 4.1.8

### Expected Behavior

Everything works as before, no error message in /var/log/daemon.log 

### Observed Behavior

After the update, the error in the attached file appears in /var/log/daemon.log every 10 to 13 seconds.
[20241216 Netbox 4.1.8 Error.log](https://github.com/user-attachments/files/18161946/20241216.Netbox.4.1.8.Error.log)

**Comment by candlerb** [comment]
Can you show the output of `/opt/netbox/venv/bin/pip freeze`?

And what version of redis server are you running?

**Comment by sbdiun** [comment]
Hello,
it turned out that although the current Redis version 7.2.6 was installed via Ansible, the SystemD daemon always started the old version 2.8.24. 
Up to 4.0.11 this did not lead to any errors, but now with 4.1.8 it does. With a manually started Redis 7.2.6 there were no more errors.
It is therefore an error in the Ansible script that needs to be corrected so that the newly installed version also runs as a service.

This actually closes the discussion. Sorry for the ticket.
